internal/resp_handler.go

```
@@ -27,20 +29,20 @@ func set(args []KVCmd, m *map[string]string) KVCmd {
 	return KVCmd{Typ: "string", Str: "OK"}
 }
 
-func get(args []KVCmd, m *map[string]string) KVCmd {
+func get(args []KVCmd, m *sync.Map) KVCmd {
 	if len(args) != 1 {
 		return KVCmd{Typ: "error", Str: "ERR wrong number of arguments for 'set' command"}
 	}
 
 	key := args[0].Bulk
 	m1 := *m
```

golangci-lint (1.62.2)

copylocks: assignment copies lock value to m1: sync.Map contains sync.Mutex

(govet)